cmd/secret/cmd/root_test.go

```diff
@@ -211,10 +210,35 @@ func TestRoot_Server(t *testing.T) {
 
 			resp, err = client.Do(req)
 			require.NoError(t, err)
-			respBody, err := ioutil.ReadAll(resp.Body)
+			_, err = ioutil.ReadAll(resp.Body)
+			require.NoError(t, err)
+			require.EqualValues(t, http.StatusOK, resp.StatusCode)
+			require.NoError(t, resp.Body.Close())
+		})
+		t.Run("middleware check success", func(t *testing.T) {
+			ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+			defer cancel()
+			expectedSipherKey := "key value"
+
+			port := createAndExecuteCliCommand(ctx)
+			middlewarePort := port + "/ping"
+			defer func() {
+				require.NoError(t, os.Remove(path))
+			}()
+
+			client := http.Client{Timeout: time.Second}
+			req := httptest.NewRequest(http.MethodGet, "http://localhost:"+middlewarePort, nil)
```

Comment:
It reads quite not right because port is a number and `/ping` is a route.
```suggestion
			defer func() {
				require.NoError(t, os.Remove(path))
			}()

			client := http.Client{Timeout: time.Second}
			req := httptest.NewRequest(http.MethodGet, fmt.Sprintf("http://localhost:%d/ping",port), nil)
```